sys/crypto/Makefile

```diff
@@ -1 +1,7 @@
+ifeq (, ${RIOT_CHACHA_PRNG_DEFAULT})
+    RIOT_CHACHA_PRNG_DEFAULT := $(shell jot -w '0x%08xull,' -r 16 0 0x7ffffffe)
```

Comment:
Please add the debian package name of ```jot``` to:
* dist/tools/travis-scripts/get-pkg-list.py
* dist/Dockerfile
* riotdocker repo (in a separate PR)

<offtopic>
We should probably create a proper list of the programs needed to build
RIOT in the very near future. Not everyone has tools like jot installed...</offtopic>